test/parallel/test-options-binding.js

```diff
@@ -0,0 +1,16 @@
+// Flags: --expose-internals
+'use strict';
+
+require('../common');
+const { internalBinding, primordials } = require('internal/test/binding');
+const {
+  SafeMap,
+} = primordials;
+
+const options = internalBinding('options');
+const assert = require('assert');
+
+assert(
+  options.getOptions() instanceof SafeMap,
```

Comment:
Doesn't `getOptions()` return an object?

https://github.com/nodejs/node/blob/fef21282ef360fc61254b460f049a9e9140258bd/lib/internal/options.js#L3-L4

```suggestion
  options instanceof SafeMap,
```

Reply:
It indeed does: <https://github.com/nodejs/node/blob/dd0b7a13d48053c7d46f697de0ec4d3b1b006d1e/src/node_options.cc#L1020-L1026>